### PR TITLE
[TASK] Adjust .gitattributes export behavior

### PR DESCRIPTION
**Pull request by longwave**
The release package contains an unnecessary file.

### CODE REVIEW

**Comment by longwave** [comment]
See #55

**Comment by ohader** [comment]
It looks correct, AppVeyor has a couple of hickups (which I did not analyze in detail)...

**Comment by ohader** [comment]
Back-port in #57

**Comment by ohader** [comment]
@longwave Thanks for your changes. Why did you close previous pull-requests, it looked okay to me. Are there any aspects I'm currently missing? Thanks in advance for your feedback!

**Comment by longwave** [comment]
Sorry, I saw the Appveyor builds failing and I thought it was because of this change it couldn't find the config file, maybe I was wrong though.

**Comment by ohader** [comment]
@longwave Alright & no problem at all... restarting AppVeyor helped in the past...
For me it would be okay to merge these changes and tag new v2.1.4 and v3.1.4 versions of the package.

File: .gitattributes [hunk 1]
```diff
@@ -1,8 +1,11 @@
+/_config.yml
```
Comment (longwave) [inline .gitattributes]: Did you intend to set `export-ignore` here?

Comment (ohader) [reply .gitattributes]: It's available in `master` only... shouldn't do any harm in `v2` branch, does it?

**Pull request merged**